hack/kind-cluster.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+set -o errexit
+
+KIND_CLUSTER_NAME="${KIND_CLUSTER_NAME:-kind}"
+KIND_VERSION=""
```

Comment:
let's call this "KIND_NODE_VERSION" or "NODE_VERSION" or "KUBERNETES_VERSION" because this is not the version of kidn we are running